sentinel-adapter/sentinel-api-gateway-adapter-common/src/main/java/com/alibaba/csp/sentinel/adapter/gateway/common/rule/GatewayFlowRule.java

```diff
@@ -25,13 +25,24 @@
  * @since 1.6.0
  */
 public class GatewayFlowRule {
+    /**间隔单位*/
+    /**0-秒*/
+    public static final int INTERVAL_UNIT_SECOND = 0;
+    /**1-分*/
+    public static final int INTERVAL_UNIT_MINUTE = 1;
+    /**2-时*/
+    public static final int INTERVAL_UNIT_HOUR = 2;
+    /**3-天*/
+    public static final int INTERVAL_UNIT_DAY = 3;
 
     private String resource;
     private int resourceMode = SentinelGatewayConstants.RESOURCE_MODE_ROUTE_ID;
 
     private int grade = RuleConstant.FLOW_GRADE_QPS;
     private double count;
     private long intervalSec = 1;
+    private Long interval;
```

Comment:
This may bring breaking changes (and also not necessary). Refer https://github.com/alibaba/Sentinel/pull/869#discussion_r298020134